qa/rpc-tests/zkey_import_export.py

```diff
@@ -169,16 +167,25 @@ def find_imported_key(node, import_zaddr):
 
         logging.info("Sending post-import txns...")
         for amount in amounts[4:]:
-            alice_to_bob(amount)
+            z_send(alice, alice_zaddr, bob_zaddr, amount)
 
         verify_utxos(bob, amounts, bob_zaddr)
         verify_utxos(charlie, amounts, ipk_zaddr)
         verify_utxos(charlie, amounts, ipk_zaddr2)
 
-        # TODO: Reproduce #1936
-        # Generate zaddr, receive ZEC. Send ZEC back out.
-        # z_import onto second node (blockchain rescan, default or True?)
-        # See if already-spent balance from first node is listed as unspent on 2nd
+        # Reproduce zombie balance reported in #1936
+        # At generated zaddr, receive ZEC, and send ZEC back out. bob -> alice
+        for amount in amounts[:2]:
+            z_send(bob, bob_zaddr, alice_zaddr, amount)
+        # z_import onto new node "david" (blockchain rescan, default or True?)
+        david.z_importkey(privkey)
+        d_ipk_zaddr = find_imported_key(david, bob_zaddr)
+        assert_equal(bob_zaddr, d_ipk_zaddr)
+        # Bug: Check if already-spent balance from bob is unspent for david
+        # UTXOs should equal amounts[2:], not amounts
+        verify_utxos(david, amounts, d_ipk_zaddr)
```

Comment:
This is the line the test is failing on.

Reply:
Yep, I think this is the user reported bug I was trying to replicate. Forgot to make this an assertion though, which allows the test to pass while still replicating the issue.